docs/concepts/ViewingKey.md

```diff
@@ -0,0 +1,3 @@
+# Viewing Keys
+
+Each shielded address corresponds to a single **viewing key**. Viewing key allows you to disclose your *complete* transaction history of a shielded address without leaking the spending secrets. This is useful in both accounting and regulation compliance. 
```

Comment:
tweaked slightly for clarity:

Each shielded address corresponds to a single **viewing key**. A viewing key allows you to disclose a shielded address's complete spending history without leaking its spending secret. This is useful for both accounting and regulation compliance.